README.md

```diff
@@ -1,3 +1,8 @@
+# **THIS PROJECT HAS MOVED**
+This project has moved to [Gitlab](https://gitlab.com/yale-sdmp/reservations).
```

Comment:
I would also display the URL (not just a link) for users to copy/paste.

Reply:
Changed